tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_migrated_valid_invalid.py

```diff
@@ -0,0 +1,288 @@
+            EOFException.MISSING_TYPE_HEADER,
+            id="EOF1I3540_0038",
+        ),
+        pytest.param(
+            # Unknown section ID (at the beginning)
+            bytes.fromhex("ef000106000101000402000100010400000000000000fe"),
+            EOFException.MISSING_TYPE_HEADER,
+            id="EOF1I3540_0039",
+        ),
+        pytest.param(
+            # Unknown section ID (at the beginning)
+            bytes.fromhex("ef0001ff000101000402000100010400000000000000fe"),
+            EOFException.MISSING_TYPE_HEADER,
+            id="EOF1I3540_0040",
+        ),
+        pytest.param(
+            # Unknown section ID (after types section)
+            bytes.fromhex("ef000101000405000102000100010400000000000000fe"),
+            EOFException.MISSING_CODE_HEADER,
+            id="EOF1I3540_0041",
+        ),
+        pytest.param(
+            # Unknown section ID (after types section)
+            bytes.fromhex("ef000101000406000102000100010400000000000000fe"),
+            EOFException.MISSING_CODE_HEADER,
+            id="EOF1I3540_0042",
+        ),
+        pytest.param(
+            # Unknown section ID (after types section)
+            bytes.fromhex("ef0001010004ff000102000100010400000000000000fe"),
+            EOFException.MISSING_CODE_HEADER,
+            id="EOF1I3540_0043",
+        ),
+        pytest.param(
+            # Unknown section ID (after code section)
+            bytes.fromhex("ef000101000402000100010500010400000000000000fe"),
+            EOFException.MISSING_DATA_SECTION,
+            id="EOF1I3540_0044",
+        ),
+        pytest.param(
+            # Unknown section ID (after code section)
+            bytes.fromhex("ef000101000402000100010600010400000000000000fe"),
+            EOFException.MISSING_DATA_SECTION,
+            id="EOF1I3540_0045",
+        ),
+        pytest.param(
+            # Unknown section ID (after code section)
+            bytes.fromhex("ef00010100040200010001ff00010400000000000000fe"),
+            EOFException.MISSING_DATA_SECTION,
+            id="EOF1I3540_0046",
+        ),
+        pytest.param(
+            # Unknown section ID (after data section)
+            bytes.fromhex("ef000101000402000100010400000500010000000000fe"),
+            EOFException.MISSING_TERMINATOR,
+            id="EOF1I3540_0047",
+        ),
+        pytest.param(
+            # Unknown section ID (after data section)
+            bytes.fromhex("ef000101000402000100010400000600010000000000fe"),
+            EOFException.MISSING_TERMINATOR,
+            id="EOF1I3540_0048",
+        ),
+        pytest.param(
+            # Unknown section ID (after data section)
+            bytes.fromhex("ef00010100040200010001040000ff00010000000000fe"),
+            EOFException.MISSING_TERMINATOR,
+            id="EOF1I3540_0049",
+        ),
+    ],
+)
+def test_migrated_valid_invalid(
+    eof_test: EOFTestFiller,
+    eof_code: Container | bytes,
```

Comment:
We could accept `Container | str` here instead of bytes, and remove whitespaces and do the `bytes.fromhex` inside the function:
```python
if type(eof_code) is str:
    eof_code = bytes.fromhex(eof_code.replace(" ", ""))  # type: ignore
```

Reply:
I will convert everything into `Container` only. However, I noticed that a `pytest.param` with `Cointainer` type produces very long test names... is this expected?

Reply:
Not really, let me push a quick fix for this.